shared/js/background/i18n.js

```diff
@@ -0,0 +1,9 @@
+import browser from 'webextension-polyfill'
+
+export function getUserLocale () {
+    if (!browser?.i18n) {
+        return 'en'
+    }
+    // returns browser locale with country suffix removed
+    return browser.i18n.getUILanguage().slice(0, 2)
```

Comment:
This doesn't handle the `nn` Norwegian case (it defaults to `en`), so perhaps we should map `nn` to `nb`?  @GioSensation might have a better sense of whether this make sense.

Reply:
From this [Stack Overflow answer](https://linguistics.stackexchange.com/questions/36784/norwegian-translation-codes-no-nn-nb-which-to-use-on-a-website) it looks like mapping `nn` and `no` to `nb` is reasonable. I added special-case handling to do that now.